### PR TITLE
Add missing data to session

### PR DESCRIPTION
**Pull request by babkenmes**
## Description

Fixes (#1540)

This adds associated_user_scope and associated_user which is stated in Shopify [documentation](https://shopify.dev/tutorials/authenticate-with-oauth#step-3-confirm-installation).

## Type of change

- [x] @shopify/koa-shopify-auth Patch: Bug (non-breaking change which fixes an issue)expected)

## Checklist

- [x] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

**Comment by babkenmes** [comment]
I need help on prettier/prettier I can not get it to build on travis

File: packages/koa-shopify-auth/src/auth/create-oauth-callback.ts [hunk 1]
```diff
@@ -53,11 +53,15 @@ export default function createOAuthCallback(config: AuthConfig) {
     }
 
     const accessTokenData = await accessTokenResponse.json();
-    const {access_token: accessToken} = accessTokenData;
-
+    const {access_token: accessToken,
+      associated_user_scope: associatedUserScope,
+      associated_user: associatedUser} = accessTokenData;
```
Comment (tanema) [inline packages/koa-shopify-auth/src/auth/create-oauth-callback.ts:58]:
```suggestion
    const {
      access_token: accessToken,
      associated_user_scope: associatedUserScope,
      associated_user: associatedUser
    } = accessTokenData;
```
I think prettier is looking for something like this. I am not sure why it is only failing on a single version though.

Comment (babkenmes) [reply packages/koa-shopify-auth/src/auth/create-oauth-callback.ts]: Still fails

File: packages/koa-shopify-auth/src/auth/create-oauth-callback.ts [hunk 2]
```diff
@@ -53,11 +53,17 @@ export default function createOAuthCallback(config: AuthConfig) {
...
 
     const accessTokenData = await accessTokenResponse.json();
-    const {access_token: accessToken} = accessTokenData;
-
+    const {
+      access_token: accessToken,
+      associated_user_scope: associatedUserScope,
+      associated_user: associatedUser
+    } = accessTokenData;
+
```
Comment (tanema) [inline packages/koa-shopify-auth/src/auth/create-oauth-callback.ts:61]:
```suggestion
    const {
      access_token: accessToken,
      associated_user_scope: associatedUserScope,
      associated_user: associatedUser,
    } = accessTokenData;

```

Yup it still says here that these things need to change, a comma on line 59 and extra spaces on line 61.

**Comment by tanema** [comment]
@TheMallen can you take a look and also assist in merging? I am not that familiar with the changelog process.

**Comment by mkevinosullivan** [comment]
Thank you for contributing to the ongoing improvement of this library - we really appreciate it.

In order to optimize the support on this particular piece of functionality, we have moved it to its own repo.  As a result, I’m closing this PR because the underlying source has been moved to a dedicated new repo; `Shopify/koa-shopify-auth` (https://github.com/Shopify/koa-shopify-auth).

If you would like the PR to be considered for review, please re-create the change against the new `Shopify/koa-shopify-auth repo` and open a new PR there.

Thank you!

**Pull request closed without merging**